src/app/components/FinanceV2Components/RowTable/TableBody/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const TableBody: React.FC = ({ children }) => (
+  <tbody className="tw-border-t tw-border-b">
+    <tr className="tw-h-16">{children}</tr>
+  </tbody>
+);
```

Comment:
`tr` here means that table must have only one row, that's correct?

Reply:
Exactly but in this case it's intentional.